pkg/cmd/bootstrap.go

```diff
@@ -432,8 +432,26 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 func nextSteps() {
 	log.Success("Bootstrapped OpenShift resources successfully\n\n",
 		"Next Steps:\n",
-		"Please refer to https://github.com/redhat-developer/kam/tree/master/docs to get started.",
+		"Please refer to https://github.com/redhat-developer/kam/tree/master/docs to get started.\n\n",
 	)
+	log.Info(" WARNING: Generated secrets are not encrypted.\n",
+		"  To use Bitnami Sealed Secrets:\n",
```

Comment:
Since we're planning to document multiple ways to manage secrets, would it be better to directly add links to the doc instead of showing steps in the command output. 
Example:
```yaml
............
............
WARNING: Generated secrets are not encrypted. For more information on secret management refer: https://github.com/redhat-developer/kam/docs/secrets

Reply:
I like your suggestion. Also, I think the external secrets set up will be more involved.